src/EnergyPlus/MixedAir.cc

```diff
@@ -4498,6 +4500,12 @@ void VentilationMechanicalProps::CalcMechVentController(
                 if (this->SystemOAMethod == SOAM_ProportionalControlSchOcc || this->SystemOAMethod == SOAM_ProportionalControlDesOcc ||
                     this->SystemOAMethod == SOAM_ProportionalControlDesOARate) {
                     SysOA = SysOA / SysEv;
+                } else if (this->SystemOAMethod == SOAM_VRP) {
+                    // Limit system OA to design OA minimum flow rate, as per ASHRAE Guideline 36-2018 Section 5.16.3.1
+                    if (state.dataSize->SysSizingRunDone) { // If no system sizing run is done (i.e. no Sizing:System) the design outdoor air flow
+                                                            // rate is not known
+                        SysOA = min(SysOAuc / SysEv, state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).DesOutAirVolFlow);
+                    }
```

Comment:
Ventilation optimization system outdoor air flow cap. We're only capping it when a system sizing run has been performed (i.e. design outdoor air flow rate is known through calculation or through user input).

Reply:
Here's an example of results before and after the implementation of the cap. In this example, one of the air terminal has its minimum air flow rate scheduled. Depending on the hourly schedule value, the system ventilation efficiency can be lower than the design value which triggers an increase in system outdoor air which is not wanted.

![image](https://user-images.githubusercontent.com/5149279/127234774-7300bcef-13e4-4d6d-bb4c-176480f0c6e1.png)